SConstruct

```diff
@@ -16,6 +16,10 @@ AddOption('--prefix', dest='prefix',
                       metavar='DIR',
                       help='installation prefix')
 
+AddOption('--video-keyframes', action='append_const', dest='cflags',
+                      const='-DVIDEO_KEYFRAMES',
+                      help= 'Build KeyFrame Optimazations for Video (Ubuntu 18)')
```

Comment:
Typo in Optimizations.
(Note there is also a (different) typo in the commit message.)